libs/hdf-converters/sample_jsons/asff_mapper/example-3-layer-overlay_hdf.json

```diff
@@ -37640,5 +37640,6 @@
       ],
       "sha256": "6ae19852d015cc96535544c069450b1d3a92f6ccd66f20b283f187f0f8f4b733"
     }
-  ]
+  ],
+  "passthrough": {}
```

Comment:
have you recreated all of these passthrough objects based off of the changes specified earlier?

Reply:
Yes, they have been recreated and are generated as expected. The only changes are the contents of each subdivision as the removal of escape character handling which lowered the character limit based off escape character count.

Reply:
Cool